insights/client/archive.py

```diff
@@ -250,7 +253,7 @@ def cleanup_previous_archive(self):
         '''
         Used at the start, this will clean the temporary directory of previous killed runs
         '''
-        for file in glob.glob('/var/tmp/insights-archive-*'):
+        for file in glob.glob('/var/tmp/insights-archive/insights-archive-*'):
```

Comment:
Wouldn't it make sense to use `constants.insights_archive_tmp_path` here instead of the hardcoded string `/var/tmp/insights-archive`?

Reply:
sure! I missed this one

Reply:
As this constant does not exist, I would prefer to create one in a separate pull request. But that’s just my obsession with unhealthily atomic changesets.